graphql/src/main/java/io/micronaut/configuration/graphql/GraphQLConfiguration.java

```diff
@@ -41,7 +41,7 @@ public class GraphQLConfiguration implements Toggleable {
     /**
      * The configuration name whether GraphQL is enabled.
      */
-    public static final String ENABLED = PREFIX + ".enabled";
+    public static final String ENABLED_CONFIG = PREFIX + ".enabled";
```

Comment:
why do we need to change this?  If we don't need it, lets change it to avoid unnecessary breaking changes.